### PR TITLE
Add 2023 directory and ideas-list.md

### PR DESCRIPTION
**Pull request by mwengren**
Placeholder for 2023 project ideas, once collected

### CODE REVIEW

**Comment by MathewBiddle** [comment]
I guess I was thinking we should have something in the README that says how to propose a project? Something that says we're looking for projects for 2023. If you have ideas, use the issue template here (with a link to an issue using the template).

**Review by MathewBiddle — APPROVED** [review]
Looks good to me.

**Review by ocefpaf — APPROVED** [review]

File: 2023/ideas-list.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Project Ideas for 2023
+
+IOOS DMAC community's project ideas list for [Google Summer of Code 2023](https://summerofcode.withgoogle.com/programs/2023).
+
+Deadline for mentoring organization application: Feb 7 , 2023.
```
Comment (ocefpaf) [inline 2023/ideas-list.md:5]: Looks better when rendered this way.

```suggestion
Deadline for mentoring organization application: Feb 7, 2023.
```

**Comment by mwengren** [comment]
Ok, thanks.  I'm going to push another change to the template in this PR and then I'll update the README after that.   

I'd update the README at the same time, but I'm not sure what the resulting [URL](https://github.com/ioos/gsoc/issues/new?assignees=&labels=GSoC&template=gsoc-project-proposal.md&title=Project+title) will be to use to link directly to the template (seems to depend on what's in the front matter of the template)...

**Pull request merged**